src/output.rs

```diff
@@ -150,11 +150,7 @@ impl OutputType {
 
     #[cfg(feature = "paging")]
     pub(crate) fn is_pager(&self) -> bool {
-        if let OutputType::Pager(_) = self {
-            true
-        } else {
-            false
-        }
+        matches!(self, OutputType::Pager(_))
```

Comment:
This is an actual (non-trivial) use case of the `matches!` macro. I'm not strictly against using it, but I'd rather still support older Rust versions for now.